src/main/java/io/fusionauth/load/FusionAuthSearchWorker.java

```diff
@@ -25,7 +25,7 @@
 import io.fusionauth.domain.search.UserSearchCriteria;
 
 /**
- * Worker to test search.
+ * Worker to test searching for users by email address.
```

Comment:
Can't this worker be used to search by anything? It just pulls the `queryString` from the config.

Reply:
The code is building an email address based on a range provided in the configuration and then using the `${email}` replacement variable. It could be used for other search, but it is coded to search by email.

I also wanted a new worker that was ready to go to search user data.

I can revert the comment update if you'd like.

Reply:
```suggestion
 * Worker to test search.
```